mappings/servers/minelandnetwork/metadata.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "minelandnetwork",
+    "name": "Mineland Network",
+    "description": "Welcome to Mineland Network — the best Cracked Minecraft server on the planet!",
+    "addresses": [
+        "mc.mineland.net",
+        "djmahirnationtv.mineland.net",
+        "mc.djmahirnationtv.com"
+    ],
+    "primaryAddress": "mc.mineland.net",
+    "minecraftVersions": [
+        "1.7.*",
+        "1.8.*",
+        "1.9.*",
+        "1.10.*",
+        "1.11.*",
+        "1.12.*",
+        "1.16.*",
+        "1.17.*",
+        "1.18.*",
+        "1.19.*",
+        "1.20.*",
+        "1.21.*"
+    ],
+    "primaryMinecraftVersion": "1.16.5",
+    "primaryColor": "#06c2be",
+    "secondaryColor": "#e66047",
+    "primaryLanguage": "en",
+    "languages": [
+        "en",
+        "de",
+        "ru",
+        "ua"
+    ],
+    "primaryRegion": "EU",
+    "regions": [
+        "AS",
+        "EU"
+    ],
+    "gameTypes": [
+        "PvP",
+        "PvE",
+        "Factions",
+        "Minigames",
+        "SkyBlock",
+        "Survival",
+        "Open World",
+        "Prison",
+        "Creative"
+    ],
+    "website": "https://mineland.net",
+    "socials": {
+        "tiktok": "https://www.tiktok.com/@minelandnetwork",
+        "instagram": "https://www.instagram.com/minelandnetwork",
+        "discord": "https://discord.gg/mineland",
+        "youtube": "https://www.youtube.com/@mineland_network"
+    },
+    "thirdPartyAPIs": [
+        "Apollo"

```

Comment:
Only include this if the server actually uses this API
Otherwise, remove this entire section